pipeline/00-ingest.R

```diff
@@ -296,33 +296,92 @@ training_data_clean <- training_data_w_hie %>%
     across(where(bit64::is.integer64), \(x) as.numeric(x))
   ) %>%
   # Get a count of the number of sales that have occurred in the last n years
+  # and the maximum price of the most recent valid sale in the last n years,
+  # both EXCLUDING the current sale
   left_join(
     left_join(
       training_data %>%
         select(meta_pin, meta_sale_document_num, meta_sale_date),
       training_data %>%
         filter(!sv_is_outlier) %>%
-        select(meta_pin, meta_sale_date),
+        select(meta_pin, meta_sale_date, meta_sale_price) %>%
+        mutate(meta_sale_price = as.numeric(meta_sale_price)),
```

Comment:
For whatever reason sale price comes out of Athena as int64, and doing math with that type in R is always a bit finnicky, so forcing a conversion here just to be safe.